src/Microsoft.DocAsCode.MarkdownLite/JsonRenderer.cs

```diff
@@ -0,0 +1,310 @@
+        {
+            var content = StringBuffer.Empty;
+
+            // header
+            var headerContent = StringBuffer.Empty;
+            for (int i = 0; i < token.Header.Length; i++)
+            {
+                var childContent = StringBuffer.Empty;
+                foreach (var item in token.Header[i].Content.Tokens)
+                {
+                    childContent += renderer.Render(item);
+                }
+                headerContent += this.Insert("headerItem", childContent.ToString(), Type.Child);
+            }
+            content += this.Insert("Header", headerContent.ToString(), Type.Child);
+
+            // Body
+            var bodyContent = StringBuffer.Empty;
+            for (int i = 0; i < token.Cells.Length; i++)
+            {
+                var rowContent = StringBuffer.Empty;
+                var row = token.Cells[i];
+                for (int j = 0; j < row.Length; j++)
+                {
+                    var childContent = StringBuffer.Empty;
+                    foreach (var item in row[j].Content.Tokens)
+                    {
+                        childContent += renderer.Render(item);
+                    }
+                    rowContent += this.Insert("RowItem", childContent.ToString(), Type.Child);
+                }
+                bodyContent += this.Insert("Row", rowContent.ToString(), Type.Child);
+            }
+            content += this.Insert("Body", bodyContent.ToString(), Type.Child);
+            return this.Insert("Table", content.ToString(), Type.Child);
+        }
+
+        public virtual StringBuffer Render(IMarkdownRenderer renderer, MarkdownNonParagraphBlockToken token, MarkdownBlockContext context)
+        {
+            var childContent = StringBuffer.Empty;
+            foreach (var item in token.Content.Tokens)
+            {
+                childContent += renderer.Render(item);
+            }
+            return this.Insert("NonParagraph", childContent.ToString(), Type.Child);
+        }
+
+        #endregion
+
+        #region Inline
+
+        public virtual StringBuffer Render(IMarkdownRenderer renderer, MarkdownEscapeInlineToken token, MarkdownInlineContext context)
+        {
+            return this.Insert("Escape:" + token.Content, this.GetSize(), Type.Size);
+        }
+
+        public virtual StringBuffer Render(IMarkdownRenderer renderer, MarkdownLinkInlineToken token, MarkdownInlineContext context)
+        {
+            var childContent = StringBuffer.Empty;
+            foreach (var item in token.Content)
+            {
+                childContent += renderer.Render(item);
+            }
+            return this.Insert("Link", childContent.ToString(), Type.Child);
+        }
+
+        public virtual StringBuffer Render(IMarkdownRenderer renderer, MarkdownImageInlineToken token, MarkdownInlineContext context)
+        {
+            return this.Insert("Image", this.GetSize(), Type.Size);
```

Comment:
I am finding a way to deal with the content of token , like the code block and Text 

---
In reply to: [74873376](https://github.com/dotnet/docfx/pull/608#discussion_r74873376) [](ancestors = 74873376)